package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "blendid",
   "version": "4.4.2",
-  "description": "(formerly gulp-starter) A full featured configurable gulp asset pipeline and static site builder",
+  "description": "A full featured configurable asset pipeline and static site builder",
```

Comment:
curious about the removal of gulp...it's still using gulp

Reply:
This whole PR is picking up a decision the whole team landed on quite a while back. Idea is remove language that ties Blendid to any specific tech, so that the tool can evolve more easily. Back in the early gulp-starter days the word "Gulp" itself was a badge of using the hot newness; now what matters is the functionality